js/scenes/AccountSettings/AccountSettingsContainer.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { ImagePickerIOS } from 'react-native';
+import AccountSettings from './AccountSettings';
+import { logout } from '../../config/helpers';
+import { 
+  updateEmailField, 
+  updatePasswordField,
+  updateFullnameField,
+  updateImageUrlField
+} from '../../redux/modules/user-forms';
+
+
+class AccountSettingsContainer extends Component {
+
+  static route = {
+    navigationBar: {
+      title: 'ACCOUNT SETTINGS',
+    }
+  }
+
+  handleEmail = (event) => {
+    this.props.dispatch(updateEmailField(event));
+  }
+
+  handlePassword = (event) => {
+    this.props.dispatch(updatePasswordField(event));
+  }
+
+  handleFullname = (event) => {
+    this.props.dispatch(updateFullnameField(event));
+  }
+
+  handleImageUpload = () => {
+    ImagePickerIOS.openSelectDialog({}, imageUri => {
+      this.props.dispatch(updateImageUrlField(imageUri));
+    }, error => console.error(error));
+  }
+
+  handleLogout = () => {
+    logout()
+  }
+
+
+  render() {
+    return (
+      <AccountSettings 
+        emailField={this.props.emailField}
+        passwordField={this.props.passwordField}
+        fullnameField={this.props.fullnameField}
+        imageUrl={this.props.imageUrl}
+
+        handleEmail={(e) => {
```

Comment:
functions with one line of code don't need closing scope.

Reply:
Please ignore the AccountSettings components as it's still being worked on (included in the PR by accident during the squashing of commits).